src/pages/VoucherRedemption/VoucherRedemption.tsx

```diff
@@ -41,10 +41,10 @@ const VoucherRedemption = (props: Props) => {
         sellerID: seller_id,
         location: getLocationInfo(merchantData),
       };
-
-      dispatch({ type: SET_VOUCHER_INFO, payload: voucher });
+      dispatch({ type: VoucherConstants.SET_VOUCHER_INFO, payload: voucher });
       setLoading(false);
-    } catch {
+    } catch (e) {
+      console.log('error: ' + JSON.stringify(e));
```

Comment:
Do we have any kind of better logging we can do here other than just a console log?